features/OJP001_Relative-placement-for-elements-aggregated-to-another-element.feature

```diff
@@ -0,0 +1,11 @@
+@implementer-agreement
+@OJP
+Feature: OJP001 - Relative placement for elements aggregated to another element
+
+  Scenario: Agreement on the relative placement of IfcElements being a part of another IfcElement through the relationship IfcRelAggregates
+
+      Given A file with Schema Identifier "IFC2X3"
+      And An IfcElement
+      And A relationship IfcRelAggregates from IfcElement to IfcElement
+      Then The relative placement of that IfcElement must be provided by an IfcLocalPlacement entity
```

Comment:
Which `that` does this sentence refer to? In the sentence above, there are two `IfcElement`s ...

Reply:
I'm not going to reply here, this feature shows up here, because my PR was based on two un-merged PRs. See #27.